NARFO_BE/NARFO_BE/Controllers/ComissionsController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using NARFO_BE.Models;
+
+namespace NARFO_BE.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ComissionsController : ControllerBase
+    {
+        private readonly narfoContext _context;
+
+        public ComissionsController(narfoContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Comissions
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Comission>>> GetComission()
+        {
+            return await _context.Comission.ToListAsync();
+        }
+
+        // GET: api/Comissions/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Comission>> GetComission(short id)
+        {
+            var comission = await _context.Comission.FindAsync(id);
+
+            if (comission == null)
+            {
+                return NotFound();
+            }
+
+            return comission;
+        }
+
+        // PUT: api/Comissions/5
+        [HttpPut("{id}")]
```

Comment:
these api's really do need swagger validation because these apis are not verry descriptive of what their purpose is
you should atleast use descriptions like these
```
  /// <summary>
        /// Gets Commision by id or whatever
        /// </summary>
        /// <param name="request"></param>
        /// <returns>returns and IActionResult that contains the comissions</returns>
```